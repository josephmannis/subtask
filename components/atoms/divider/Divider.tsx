import React from 'react';
import styled from 'styled-components/native';
import Icon from '../icon/Icon';


export const Base = styled.View`
    margin: 0px 12px;
`
const Divider: React.FC = props => {
    return (
        <Base>
            <Icon size='small' type='arrow'/>
        </Base>
    )
}

export default Divider;