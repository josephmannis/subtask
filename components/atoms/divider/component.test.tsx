import React from 'react';

import renderer from 'react-test-renderer';
import Divider from './Divider';
test('Renders', () => {
    const tree = renderer.create(<Divider/>).toJSON();
    expect(tree).toMatchSnapshot()
})