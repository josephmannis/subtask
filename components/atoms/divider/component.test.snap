// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders 1`] = `
<View
  style={
    Array [
      Object {
        "alignSelf": "center",
        "marginBottom": 0,
        "marginLeft": 12,
        "marginRight": 12,
        "marginTop": 0,
      },
    ]
  }
>
  <Image
    size="small"
    source={1}
    style={
      Array [
        Object {
          "aspectRatio": 1,
          "height": 20,
          "resizeMode": "contain",
          "width": 20,
        },
      ]
    }
  />
</View>
`;
